presto-raptor/src/test/java/com/facebook/presto/raptor/systemtables/TestShardMetadataRecordCursor.java

```diff
@@ -140,10 +165,133 @@ public void testSimple()
         }
         assertEquals(actual.size(), 3);
 
+        ImmutableList.Builder<MaterializedRow> expected = ImmutableList.builder();
+        for (ShardInfo shard : shards) {
+            Slice uuid = utf8Slice(shard.getShardUuid().toString());
+            Slice hash = utf8Slice(format("%016x", shard.getXxhash64()));
+            expected.add(
+                    new MaterializedRow(
+                            DEFAULT_PRECISION,
+                            schema, table, uuid, null, shard.getUncompressedSize(), shard.getCompressedSize(), shard.getRowCount(), hash,
+                            null, null, null, null)
+            );
+        }
+        assertEquals(actual, expected.build());
+    }
+
+    @Test
+    public void testTimestampTemporalColumnWithFilter()
```

Comment:
I added more tests into `TestRaptorIntegrationSmokeTest`. However, we can not verify the actual content of each row in the shards table (e.g. shard UUID, temporal ranges, etc.). I suggest we still keep the unit tests in `TestShardMetadataRecordCursor`.